components/organisms/ToolList/nav.tsx

```diff
@@ -53,16 +49,13 @@ const Nav: React.FC<NavProps> = ({ toolList, selectedTool = "dashboard", selecte
           />
         </div>
       ))}
-      {range ? (
-        <div className="ml-auto hidden md:block">
-          <ComponentDateFilter
-            setRangeFilter={(selectedRange) => {
-              setQueryParams({ range: `${selectedRange}` });
-            }}
-            defaultRange={Number(range)}
```

Comment:
Refactor (blocking): We need to keep the `defaultRange` because if you navigate to e.g. https://deploy-preview-2573--oss-insights.netlify.app/javascript/dashboard/filter/recent?range=90, it will load the default of 30 days.

TLDR; Keep this file as it was, but add a default value if none is in the querystring, i.e.

```typescript
const { range = 30 } = router.query;
```